### PR TITLE
Update version to 2.6.0

### PR DESCRIPTION
**Pull request by laurensverhoeven**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -55,10 +55,9 @@ def get_ec_name_by_system():
         else:
             raise ValueError('Cannot determine architecture')
 
-        return 'ec-{}-{}{}'.format(
+        return 'ec-{}-{}'.format(
```
Comment (mmicu) [inline setup.py:58]: Why did you remove this?

File: setup.py [hunk 2]
```diff
@@ -94,8 +93,6 @@ def extract_tarball(url, data):
 
 def save_executables(data, base_dir):
     exe = EDITORCONFIG_CHECKER_EXE_NAME
-    if system() == 'Windows':
```
Comment (mmicu) [inline setup.py:97]: Why did you remove this?

File: setup.py (hunk 1 above)
Comment (laurensverhoeven) [reply setup.py:58]: For version 2.6.0, the files for windows lack the `.exe` in the filename that was present earlier, so the filenames are now simply consistent with the other systems. See https://github.com/editorconfig-checker/editorconfig-checker/releases/tag/2.6.0 .

File: setup.py (hunk 2 above)
Comment (laurensverhoeven) [reply setup.py:97]: See the other comment

File: setup.py (hunk 1 above)
Comment (mmicu) [reply setup.py:58]: Cool, thanks for the info.
